src/logger_dispatch_overrides.jl

```diff
@@ -24,6 +24,31 @@ preprocess(name, val::WrapperLogType, data) = push!(data, name=>val)
 Base.show(io::IO, mime::AbstractString, x::WrapperLogType) =
     Base.show(io, mime, content(x))
 
+########## For things going to LogAudio ##############################
+"""
+    TBAudio(data, samplerate)
```

Comment:
wrong doctoring? It's supposed to be for `TBAudios`

Reply:
I will change that